src/celeritas/mat/MaterialParams.cc

```diff
@@ -109,13 +129,29 @@ MaterialParams::from_import(ImportData const& data)
  */
 MaterialParams::MaterialParams(Input const& inp)
 {
+    CELER_EXPECT(!inp.elements.empty());
     CELER_EXPECT(!inp.materials.empty());
 
     ScopedMem record_mem("MaterialParams.construct");
 
-    // Build elements and materials on host.
+    // Build input data on host
     HostValue host_data;
 
+    isotope_data_ = !inp.materials.empty() ? true : false;
```

Comment:
```suggestion
    isotope_data_ = !inp.materials.empty();
```
but can't you just replace `isotope_data_` in practice with `this->num_isotopes() > 0`?

Reply:
Agreed, `num_isotopes()` provides the same functionality, so I'll remove the bool. I also realized that I accepted the wrong vscode autocompletion: that should have been `inp.isotopes`... 😬